### PR TITLE
Propagates channel status to listener

### PR DESCRIPTION
**Pull request by cooksey**
* Adds channelUsersOnline property to Session
* Implements channelFeatures property
* Adds onChannelStatusUpdated() method to SessionListener

### CODE REVIEW

File: channel-sdk/src/main/java/com/zello/channel/sdk/ChannelFeature.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.zello.channel.sdk
+
+// TODO: Document ChannelFeature
```
Comment (zello-eugene) [inline channel-sdk/src/main/java/com/zello/channel/sdk/ChannelFeature.kt:3]: Are you planning on a separate pass at adding the missing documentation later?

Comment (cooksey) [reply channel-sdk/src/main/java/com/zello/channel/sdk/ChannelFeature.kt:3]: Yes, after all features are implemented I'm going to make a pass to make sure everything is documented. I can document this now though if you'd be more comfortable that way.

File: channel-sdk/src/main/java/com/zello/channel/sdk/Session.kt [hunk 2]
```diff
@@ -322,7 +323,9 @@ class Session internal constructor(
     private fun performDisconnect(): Boolean {
         val transport = transport
         state = SessionState.DISCONNECTED
-        this.transport = null
+		channelFeatures = EnumSet.noneOf(ChannelFeature::class.java)
```
Comment (zello-eugene) [inline channel-sdk/src/main/java/com/zello/channel/sdk/Session.kt:326]: There appears to exist a mix of spaces and tabs as whitespaces. We use tabs in the rest of our Java/Kotlin codebase, would you replace the spaces with tabs? I suggest setting up the auto-formatting in AS.

Comment (cooksey) [reply channel-sdk/src/main/java/com/zello/channel/sdk/Session.kt:326]: Man, I thought I had. I'll try to get it fixed.

Comment (cooksey) [reply channel-sdk/src/main/java/com/zello/channel/sdk/Session.kt:326]: My editor was already set up to use tabs. Looks like maybe the file had previously had spaces instead?

**Pull request merged**